src/hotspot/share/gc/parallel/parallelScavengeHeap.hpp

```diff
@@ -102,7 +102,8 @@ class ParallelScavengeHeap : public CollectedHeap {
   HeapWord* allocate_old_gen_and_record(size_t word_size);
 
  protected:
-  HeapWord* allocate_new_tlab(size_t min_size, size_t requested_size, size_t* actual_size) override;
+  HeapWord* allocate_new_tlab(size_t min_size, size_t requested_size, size_t* actual_size,
+                              bool* gc_overhead_limit_was_exceeded) override;
```

Comment:
```suggestion
  HeapWord* allocate_new_tlab(size_t min_size,
                              size_t requested_size,
                              size_t* actual_size,
                              bool* gc_overhead_limit_was_exceeded) override;
```
Parameter formatting

Reply:
Fixed.